### ISSUE TITLE
PushTransaction throw exception because unknown field. But it still included in block

### ISSUE THREAD

**Issue by trongcauhcmus** [EOSEssentials/eos-java-rpc-wrapper#26]
i just meet a bug throw out an exception. because in new api version, it added field `context_free`, but the transaction still included in the network.
This is a critical bug. because in exchange trading. when user withdraw their fund. we don't know it success or not because the response from eos node was marked with an exception.
you guys should ignore unknown field in the source code to make the stable version when eos node change their api. I lost 3k$ because this bug. and we don't have any release version for this. you should warning user who use this library for stable version or they should do json parser by their self

**Comment by Fletch153** [comment]
Hi Trong, i'm sorry to hear you lost $3k.

A bug seemed to have been included which prevented longs from being deserialized. I've just added a fix for this. I've also enabled @JsonIgnoreProperties on every class which will cause any unknown fields to be ignored in the future.

In the future, it's definitely worth testing any package updates on your system in a test environment thoroughly before deploying to production, to prevent scenarios like this from occuring.
